README.md

```diff
@@ -1,135 +1,131 @@
+Any framework that can generate a build output in accordance with the App Hosting output bundle can be deployed on App Hosting.
 
-This repository is maintained by Google but is not a supported Firebase product. Issues here are answered by
-maintainers and other community members on GitHub on a best-effort basis.
+The output bundle primarily consists of a bundle.yaml file that sits inside of the .apphosting directory. This bundle.yaml contains all the ways that frameworks can configure App Hosting when users deploy their applications.
 
-[Please open issues related to Web Frameworks support in Firease CLI in the firebase-tools repository](https://github.com/firebase/firebase-tools/issues/new/choose).
+NOTE: App Hosting technically supports all all node applications, but no custom framework features will be enabled without the output bundle.
 
-## Enable framework-awareness
+## Output bundle Schema
 
-An experimental add-on to the Firebase CLI provides web framework support. To enable it, call the following:
+The output bundle all lives inside a single file:
 
 ```shell
-firebase experiments:enable webframeworks
+.apphosting/bundle.yaml
 ```
 
-## Prerequisites
-
-- Firebase CLI version 10.9.1 or later (see installation instructions [here](https://firebase.google.com/docs/cli))
-
+As long as this file exists and follows the schema App Hosting will be able to process the build properly.
 
-## Initialize Firebase Hosting
+The schema can also be found in <https://github.com/FirebaseExtended/firebase-framework-tools/blob/main/packages/%40apphosting/common/src/index.ts#L4>
 
-When you initialize Firebase Hosting it should automatically detect known Web Frameworks, if one isn't discovered
-you'll be given a list of supported frameworks to start with.
-
-```shell
-firebase init hosting
-```
-
-You should see the "source" option in your `firebase.json` rather than the traditional "public". This points to the
-root directory of your application's source code, relative to your `firebase.json`.
-
-```json
-{
-  "hosting": {
-    "source": ".",
-    "ignore": [
-      "firebase.json",
-      "**/.*",
-      "**/node_modules/**"
-    ],
-    "frameworksBackend": {
-      "region": "us-central1"
-    }
-  }
+```typescript
+interface OutputBundle {
+  version: "v1"
+  runConfig: RunConfig;
+  metadata: Metadata;
 }
 ```
 
-## Serve locally
-
-You can test your integration locally by following these steps:
+### Version
 
-1. Run `firebase emulators:start` from the terminal. This should build your app and serve it using the Firebase CLI.
-2. Open your web app at the local URL returned by the CLI (usually http://localhost:5000).
+The version represents which output bundle version is currently being used. The current version is v1.
 
-## Deploy your app to Firebase Hosting
+### RunConfig
 
-When you're ready to share your changes with the world, deploy your app to your live site:
+The runConfig fields are in charge of configuring the backend’s Cloud run deploy.
```

Comment:
If this were Devsite, I'd suggest backticks for all literals, but I'm not 100% what the convention is here.